net-dialup/freeradius-client/freeradius-client-1.1.7.ebuild

```diff
@@ -15,24 +15,30 @@ KEYWORDS="~alpha amd64 arm arm64 hppa ~ia64 ppc ppc64 sparc x86"
 
 IUSE="scp shadow static-libs"
 
-DEPEND="!net-dialup/radiusclient
-	!net-dialup/radiusclient-ng"
+DEPEND="!net-dialup/radiusclient-ng"
 RDEPEND="${DEPEND}"
 
-DOCS=( BUGS doc/ChangeLog doc/login.example doc/release-method.txt )
+DOCS=(
+	BUGS doc/ChangeLog doc/login.example doc/release-method.txt doc/instop.html
+)
+
+src_prepare() {
+	default
+	mv configure.in configure.ac
+	eautoreconf
+}
 
 src_configure() {
-	local myeconfargs=(
-		$(use_enable scp)
-		$(use_enable shadow)
+	econf \
+		$(use_enable scp) \
+		$(use_enable shadow) \
 		--with-secure-path
-	)
-	autotools-utils_src_configure
 }
 
 src_install() {
-	autotools-utils_src_install
-	dodoc README*
+	default
+	emake DESTDIR="${D}" install
 	newdoc doc/README README.login.example
-	dohtml doc/instop.html
+	rm doc/README
+	dodoc README*
```

Comment:
Why don't you add this to the `DOCS` array above?

Reply:
Because if I had `README*` portage complains `/home/alarig/Documents/git/github/alarig/gentoo/net-dialup/freeradius-client/freeradius-client-1.1.7-r1.ebuild: line 24: no match: README*`

Reply:
I just had an idea, switching the array to a string fixes it.